hack/e2e.sh

```diff
@@ -37,7 +37,7 @@ METERING_REPO_VERSION="$(basename $(find $MANIFESTS_BASE_DIR -type d | sort -r |
 METERING_RUN_TESTS_LOCALLY="${METERING_RUN_TESTS_LOCALLY:=false}"
 METERING_OLM_SUBSCRIPTION_CHANNEL="${METERING_OLM_SUBSCRIPTION_CHANNEL:="4.6"}"
 METERING_OLM_UPGRADE_FROM_SUBSCRIPTION_CHANNEL="${METERING_OLM_UPGRADE_FROM_SUBSCRIPTION_CHANNEL:="4.5"}"
-METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY="${METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY:=""}"
+METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY="${METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY:="registry.svc.ci.openshift.org/ocp/4.6:metering-ansible-operator-registry"}"
```

Comment:
Note: this image does require extra pull permissions, and we haven't mirrored this image to quay yet. I can walk anyone through the process for being able to pull this image locally, but it shouldn't be needed when creating these resources in a cluster.